src/blocks/_calendar.scss

```diff
@@ -0,0 +1,29 @@
+@import "../styles/variables";
```

Comment:
As I know it should work without this line because you import variables before calendar:
```
@import "./variables";
@import "../blocks/calendar";
@import "../blocks/days";
```